Configuration/TCA/Overrides/tt_content_plugin_haiku.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+defined('TYPO3') or die();
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
```

Comment:
use should only be used in ext_localconf.php and ext_tables.php as they are namespaced since v11. TCA files are not by now, so this may create conflicts on larger installations.